packages/server-api/package.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "server-api",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "node index.js",
+    "test": "echo testing Node server"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

```suggestion
  "license": "EPL-2.0",
```
We're licensing the Project Ricotta project under the Eclipse Public License, v2.0